node/middlewares/internals/brand.ts

```diff
@@ -8,29 +8,38 @@ import {
   routeFormatter,
   STORE_LOCATOR,
 } from '../../utils/internals'
+import { createTranslator } from '../../utils/messages'
 import { slugify } from '../../utils/slugify'
 
 export async function brandInternals(ctx: Context, next: () => Promise<void>) {
   const {
-    clients: { apps },
-    state,
+    clients: { apps, messagesGraphQL },
+    state: {
+      tenantInfo: { defaultLocale: tenantLocale },
+      tenantInfo,
+    },
   } = ctx
   const brand: Brand = ctx.body
   const { name, active, id } = brand
-  const brandName = slugify(name)
-  const bindings = filterStoreBindings(ctx.state.tenantInfo)
+  const bindings = filterStoreBindings(tenantInfo)
 
   if (bindings.length === 0) {
     return
   }
 
   const formatRoute = await routeFormatter(apps, PAGE_TYPES.BRAND)
+  const translate = createTranslator(messagesGraphQL)
+  const messages = [{ content: name.toLowerCase(), context: id }]
```

Comment:
Why to lower case?

Reply:
I will remove this. I thought on adding this so the translations are case insensitive but maybe it's better to remove custom logic